tests/test_registerutils.py

```diff
@@ -458,8 +487,2449 @@ def test_clean_host_file_no_empty_bottom_lines_smt_entry_is_last():
     assert m().write.mock_calls == expected_write_calls
 
 
+def test_clean_host_file_raised_exception():
+    hosts_content = ""
+    expected_cleaned_hosts = """
+# simulates hosts file containing the ipv6 we are looking for in the test
```

Comment:
this says "ipv6" but what follows are ipv4 address, meaning this duplicates the mistake that is already in the tests.